From: David Herrmann <dh.herrmann@gmail.com>
Date: Wed, 3 Jun 2015 17:53:29 +0200
Subject: [PATCH] kdbus/selftests: add build-dependencies on headers

Make sure the selftests are re-built if one of the local headers changes.

Signed-off-by: David Herrmann <dh.herrmann@gmail.com>
---
 tools/testing/selftests/kdbus/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/testing/selftests/kdbus/Makefile b/tools/testing/selftests/kdbus/Makefile
index 076f9f40566d..7ad587b3c767 100644
--- a/tools/testing/selftests/kdbus/Makefile
+++ b/tools/testing/selftests/kdbus/Makefile
@@ -34,7 +34,7 @@ all: kdbus-test
 
 include ../lib.mk
 
-%.o: %.c
+%.o: %.c kdbus-enum.h kdbus-test.h kdbus-util.h
 	$(CC) $(CFLAGS) -c $< -o $@
 
 kdbus-test: $(OBJS)
